contents/handbook/engineering/support-hero.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Support Hero
+sidebar: Handbook
+showTitle: true
+---
+
+Every week, we assign one person to be the "Support Hero." If this is you this week, congratulations! Support hero is an intense but super fun week where you get to talk to a bunch of users, get the statisfaction of helping them out and contribute to a lot of different parts of our system.
+
+All other work takes a back seat while you're doing support, so don't plan on doing any 'normal' work.
+
+## Availability 
+
+You should work your 'normal' hours during this week. Right now people in other timezones will jump in ad-hoc to catch messages that fall outside of the Support Hero's hours. Just make sure that as support hero you are doing the bulk of the debugging/fixing.
+
+If you are planning on taking a day off or you won't be available, please find someone to swap with. 
+
+## Rota
+
+You can view the Support Hero rota [in PagerDuty here](https://posthog.pagerduty.com/schedules#PPLGE4G). 
+
+## Channels
+
+There are a couple of channels that customer requests come in so make sure you keep an eye on all of them:
+
+- [PostHog Users's Slack](https://posthog.com/slack), specifically #_customer_support, where all messages will come in from the other channels
+- GitHub issues, with [the main repo](https://github.com/posthog/posthog/issues) being the most important one
+- Sentry issues, either [directly](https://sentry.io/organizations/posthog/issues/?project=1899813) or in #sentry in our main Slack
+
+## Communication
+
+As an engineer, when a question comes in your first instinct is to give them an answer as quickly as possible. That means we often forget pleasantries, or will ignore a question until we've found the answer. Hence the following guidelines:
+
+- Always respond to a question within a reasonable timeframe during your working day (<15 minutes is great, <1 hour is okay, anything over a day is too late)
+  - If you're ready to look into the issue and you think it might take a while/require a fix, just mention that and say you'll get back to them
+  - If you have no idea how to answer or fix their issue, @mention someone who does
```

Comment:
I propose we suggest using team slack channels (e.g. #team-platform for infra/deployment questions) instead (& calling out which one to use for what too), because:
1. Support hero might not know who they should tag if it's not in their area
2. folks who have been here longer are more likely to get tagged, which creates uneven workload where more junior folks could have jumped in to help && also longer response times as we're waiting on a single person's availability here.